module/metrics.go

```diff
@@ -94,13 +94,13 @@ type CleanerMetrics interface {
 }
 
 type CacheMetrics interface {
-	// report the total number of cached items
+	// CacheEntries report the total number of cached items
 	CacheEntries(resource string, entries uint)
-	// report the number of times the queried item is found in the cache
+	// CacheHit report the number of times the queried item is found in the cache
 	CacheHit(resource string)
-	// report the number of items the queried item is not found in the cache, nor found in the database
+	// CacheNotFound report the number of items the queried item is not found in the cache, nor found in the database
```

Comment:
s~(number of) items~$1 times~ ? :-)

Reply:
In the other comment, I requested the original developer @zhangchiqing to have a quick look (@jordanschalm the other original developer is on vacation AFAIK).

Reply:
https://github.com/onflow/flow-go/pull/2093/commits/2cc141ff142bd3b3550643a33829df81db8cac8e